pep257.py

```diff
@@ -493,13 +494,16 @@ def lines(self):
     def __str__(self):
         self.explanation = '\n'.join(l for l in self.explanation.split('\n')
                                      if not is_blank(l))
-        template = '%(filename)s:%(line)s %(definition)s:\n        %(message)s'
-        if self.source and self.explain:
-            template += '\n\n%(explanation)s\n\n%(lines)s\n'
-        elif self.source and not self.explain:
-            template += '\n\n%(lines)s\n'
-        elif self.explain and not self.source:
-            template += '\n\n%(explanation)s\n\n'
+        if not Error.fmt:
+            template = '%(filename)s:%(line)s %(definition)s:\n        %(message)s'
```

Comment:
Travis is complaining because of this line. My proposal for improvement:
```python
            template = ('%(filename)s:%(line)s %(definition)s:\n'
                        '        %(message)s')
```